consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-configuring-topics"]
+= Configuring consumer groups in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: configuring-consumer-groups
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+Consumer groups share large data streams generated by producers from a given topic in a Kafka instance.
+Grouping consumers scales consumption to keep up with the rate of data produced.
+Consumers within a group don’t read data from the same partition, but can receive data from one or more partitions.
+
+As a developer of applications and services, you can view all the consumer groups that have access to a particular Kafka instance in {product}.
+If required, use the {product} web console to reset the offsets of consumer groups or delete consumer groups.
+
+As an alternative to using the {product} web console, you can use the `rhoas` command-line interface (CLI) or the Kafka `kafka-consumer-groups.sh` tool to review and update consumer groups.
+
+//Additional line break to resolve mod docs generation error
+
+[id="con-adding-consumer-group-id_{context}"]
+== How to add a consumer to a consumer group
+
+A group ID is _required_ for a consumer to be able to join a consumer group.
+
+When you configure a consumer client application to access Kafka, you assign a group ID to associate it with a consumer group.
+All consumers with the same group ID belong to the consumer group with that ID.
+Consumer groups connected to a Kafka instance are presented in the {product} web console.
+
+The group ID is the _name_ of the consumer group.
+Add it as a property to your consumer configuration.
+The group ID can't be null or empty.
+
+.Example group ID property configuration for a consumer
+[source,properties]
+----
+group.id=my-group-id
+----
+
+.Property attributes
+[cols="25%,75%"]
+|===
+
+h|Type
+|String
+
+h|Default value
+|None
+
+h|Supported values
+|Letters (Aa-Zz), numbers (0-9), underscores ( _ ), or hyphens ( - )
+|===
+
+If you are using Kafka bin scripts to consume messages, you can use the `kafka-console-consumer.sh` tool to create a group ID:
```

Comment:
In my previous comment, I suggested "you're using" for consistency in the tone you've set.